components/molecule/modal/src/index.js

```diff
@@ -48,7 +48,7 @@ const MoleculeModal = ({
 
   const closeModal = useCallback(
     ev => {
-      ev?.stopPropagation()
+      ev.stopPropagation()
```

Comment:
why are you removing it? anyhow, as you did not commit this under the molecule/modal scope no release would be created

Reply:
because I was getting this error
<img width="1256" alt="Captura de pantalla 2020-09-02 a las 15 54 43" src="https://user-images.githubusercontent.com/55990391/92088205-db810b00-edcc-11ea-839a-1624b9cc3ca0.png">